ui/packages/ace/lib/webpack.config.js

```diff
@@ -254,7 +254,9 @@ const test = (base, { main }) => merge.smart(base, {
 const prod = (base, { main }) => merge.smart(base, {
   mode: 'production',
   devtool: 'source-map',
-  entry: resolve(main),
+  entry: [
+    resolve(main) 
```

Comment:
@andrewkfiedler - seeing a windows build failure here:
`webpack.config.js:258:18: Trailing spaces not allowed`

Reply:
Confirmed that removing the trailing space fixed the build issue